pandas.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+"""pandas.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1d-Bn6_9MT2YvqdNjhcWlrq3UWnIlWrCA
+"""
+
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+
+# функции read_gff и read_bed6 для чтения соответствующих форматов
+
+def read_gff(file):
+    col = ['chromosome', 'source', 'type', 'start', 'end', 'score', 'stand', 'phase', 'attributes']
+    return pd.read_csv(file, comment='"', header=None, names=col, sep = '\s+').drop(index=0)
+
+def read_bed6(file):
+    col = ['chromosome', 'start', 'end', 'name', 'score', 'stand']
+    return pd.read_csv(file, comment='"', header=None, names=col, sep = '\s+').drop(index=0)
+
+df_gff = read_gff('rrna_annotation.gff')
+df_bed6 = read_bed6('alignment.bed')
+
+df_gff.head()
+
+df_bed6.head()
+
+# сокращение колонки атрибутов
+
+new_df_gff = df_gff.copy()
+df_gff['attributes'] = df_gff['attributes'].apply(lambda x: x.split('=')[-1])
+df_gff.head()
+
+# BarPlot количество рРНК каждого типа
+
+chromo = [x[1] for x in sorted([[int(x.split('_')[-1]), x] for x in df_gff['chromosome'].unique()])]
+barplot = [[], [], []]
+
+for chr in chromo:
+    barplot[0].append(len(df_gff[(df_gff['chromosome'] == chr) & (df_gff['attributes']=='16S')]))
+    barplot[1].append(len(df_gff[(df_gff['chromosome'] == chr) & (df_gff['attributes']=='23S')]))
+    barplot[2].append(len(df_gff[(df_gff['chromosome'] == chr) & (df_gff['attributes']=='5S')]))
```

Comment:
А зачем списки? У нас задание по пандасу, так что стоит использовать датафреймы
```suggestion
df_gff.groupby(['chromosome', 'attributes'], as_index=False).size()
```